cesium/internal/unary/unary_suite_test.go

```diff
@@ -20,15 +19,16 @@ import (
 )
 
 var (
-	ctx         = context.Background()
-	rootPath    = "unary-testdata"
-	fileSystems map[string]func() xfs.FS
-	cleanUp     func() error
+	ctx                  = context.Background()
+	rootPath             = "unary-testdata"
+	fileSystems, cleanUp = testutil.FileSystems()
 )
 
 func TestUnary(t *testing.T) {
 	RegisterFailHandler(Fail)
-	fileSystems, cleanUp = testutil.FileSystems()
 	RunSpecs(t, "Unary Suite")
-	Expect(cleanUp()).To(Succeed())
 }
+
+var _ = BeforeSuite(func() { Expect(fileSystems).To(HaveLen(2)) })
```

Comment:
Is this assertion useful?

Reply:
Yes, if there is an error in creating the osFS `tempDir`, it would not get added to the map. we would know if that went wrong this way.